mandb.te

```diff
@@ -37,24 +37,12 @@ dev_search_sysfs(mandb_t)
 domain_use_interactive_fds(mandb_t)
 
 files_read_etc_files(mandb_t)
+files_search_pids(mandb_t)
 
 miscfiles_manage_man_cache(mandb_t)
 miscfiles_read_man_pages(mandb_t)
 miscfiles_read_localization(mandb_t)
 
-ifdef(`distro_debian',`
-	optional_policy(`
-		apt_exec(mandb_t)
-		apt_read_db(mandb_t)
-	')
-
-	optional_policy(`
-		dpkg_exec(mandb_t)
-		dpkg_read_db(mandb_t)
-		userdom_dontaudit_search_user_home_dirs(mandb_t)
-	')
```

Comment:
mmmh

Post install scripts are executed in dpkg_script_t (the maintainer script) now I think and not dpkg (or even apt, higher in the block). @cgzones did you check that too?